torch/nn/modules/loss.py

```diff
@@ -1121,6 +1125,91 @@ def forward(self, input: Tensor, target: Tensor) -> Tensor:
                                ignore_index=self.ignore_index, reduction=self.reduction)
 
 
+class CrossEntropyLossWithSoftLabels(_Loss):
+    r"""This criterion combines :class:`~torch.nn.LogSoftmax` and :class:`~torch.nn.KLDivLoss` in
```

Comment:
If anyone else is interested in reading the rendered docs: https://14108352-65600975-gh.circle-artifacts.com/0/docs/generated/torch.nn.CrossEntropyLossWithSoftLabels.html?highlight=crossentropy#torch.nn.CrossEntropyLossWithSoftLabels